db/seeds.rb

```diff
@@ -47,3 +49,79 @@ class SeedAdminENVError < StandardError
 business_categories.each { |b_category| BusinessCategory.find_or_create_by(name: b_category) }
 BusinessCategory.find_or_create_by(name: 'Sociala tjänstehundar', description: 'Terapi-, vård- & skolhund dvs hundar som jobbar tillsammans med sin förare/ägare inom vård, skola och omsorg.')
 BusinessCategory.find_or_create_by(name: 'Civila tjänstehundar', description: 'Assistanshundar dvs hundar som jobbar åt sin ägare som service-, signal, diabetes, PH-hund mm')
+
+if Rails.env.development? || Rails.env.staging?
+  r = Random.new
+  USERS = 40
+  regions = Region.all.to_a # regions are created in a migration file
+  num_regions = regions.size
+
+  # Create users
+  users = []
+
+  USERS.times do
+    users << User.create(email: FFaker::InternetSE.free_email,
+                         password: 'password')
+  end
+
+  puts "Users created: #{User.all.count}"
+
+  # Create membership application for some users
+  # (two rounds - so some of the users have more than one application)
+
+  business_categories = BusinessCategory.all.to_a
+  num_cats = business_categories.size
+
+  applications = []
+
+  2.times do
+    r.rand(1..USERS).times do
+
+      company_number = r.rand(1000000000..9999999999).to_s
+
+      ma = MembershipApplication.new(first_name: FFaker::NameSE.first_name,
+                                     last_name: FFaker::NameSE.last_name,
+                                     contact_email: FFaker::InternetSE.free_email,
+                                     company_number: company_number,
+                                     status: 'Pending',
+                                     user: users[r.rand(0..USERS-1)])
+      idx1 = r.rand(0..num_cats-1)
+      ma.business_categories << business_categories[idx1]
+      idx2 = r.rand(0..num_cats-1)
+      ma.business_categories << business_categories[idx2] if idx2 != idx1
+
+      ma.save(validate: false)
+
+      applications << ma
+    end
+  end
+
+  puts "Applications created: #{MembershipApplication.all.count}"
+
+  # Accept some of the membership applications
+  r.rand(1..applications.size).times do
+    ma = applications[r.rand(0..(applications.size-1))]
+
+    next if ma.status == 'Godkänd'
```

Comment:
put 'Godkand' into a constant:  (1) it's used more than 2 places (I start thinking about doing this when I have 3 or more places), and (2) it will make this easier to change.  (We know we'll be changing this quite soon with the state changes we're doing.)